docs/language-server/README.md

```diff
@@ -11,14 +11,19 @@ These documentations are intended for the developers who wish to understand the
 server implementation and for developers who are willing to write new language server features.
 
 ## Features
-* [Ballerina Language Server Features](Features.md)
+* [Completions and Code Actions](Features.md)
+
+## Document Event Sync PubSub Model
+A mechanism to get notified on 
```

Comment:
How about we add a comma at the end?
```suggestion
A mechanism to get notified on, 
```

Reply:
I had forgotten to finish this sentence. Fixed it.